src/tests/ftest/server/metadata.py

```diff
@@ -164,18 +163,18 @@ def test_metadata_addremove(self):
         Use Cases:
             ?
 
-        :avocado: tags=metadata,metadata_free_space,nvme,small
+        :avocado: tags=metadata,metadata_free_space,nvme,small,hw,full_regression
```

Comment:
(pylint-line-too-long) Line too long (81/80)

Reply:
You can avoid this by doing the following:
:avocado: tags=metadata,metadata_free_space,nvme,small,hw
:avocado: tags=full_regression

Reply:
Pardon the emoji. ':avocado:' automatically got translated to the emoji.